documentation/content/visiting.html

```diff
@@ -481,6 +457,43 @@ <h3 id="queueing">Queueing</h3>
 
 
 
+<h2 id="visitor-processor-types">Visitor processor types</h2>
+<table class="table">
+  <thead></thead><tbody>
+    <tr>
+      <th>Dump visitor</th>
+      <td>
+        The most commonly used visitor processor type is the dump visitor.
+        All it does is to send the read documents on to some external target
+        specified by the visitor.
+        Using the command line tool <em>vespa-visit</em>
+        the default is to just send the documents back to the client,
+        and have them printed to stdout.
+        The dump visitor is used to implement reprocessing.
+        Typically, using a messagebus route,
+        that will send the documents through the document processing cluster
+        and then back to the content cluster.
+        Migration of documents from one cluster to another
+        is also implemented using a dump visitor.
+      </td>
+    </tr><tr>
+      <th style="white-space: nowrap">Streaming search visitor</th>
+      <td>
+        Another visitor processor is the streaming search visitor. This is run on the search containers,
+        making it transparent whether search results were created from streaming search,
+        or through indexed search in proton.
+      </td>
+    </tr><tr>
+      <th>Statistics visitor</th>
+      <td>
+        There is also a statistics visitor that just generate some statistics of stored data
```

Comment:
I'm not confident this visitor actually works these days. Do we have any system tests for it or known users? If not, we should probably remove it from the list. Especially since it's not documented how to actually use the thing.

Reply:
less is more! removed